core/templates/dev/head/exploration_editor/main_tab/StateResponses.js

```diff
@@ -306,6 +308,26 @@ oppia.controller('StateResponses', [
         return choiceIndices.every(function(choiceIndex) {
           return handledAnswersArray.indexOf(choiceIndex) !== -1;
         });
+      } else if (interactionId === 'ItemSelectionInput') {
+        var numChoices = $scope.getAnswerChoices().length;
+        for (var i = 0; i < answerGroups.length; i++) {
+          var ruleSpecs = answerGroups[i].rule_specs;
```

Comment:
Thanks @wxyxinyu I'll keep trying this weekend. It's been a really bad couple of weeks so I set this aside but I'll work to get it done by Sunday.